virttest/libvirt_xml/nodedev_xml.py

```diff
@@ -91,17 +91,60 @@ def get_sysfs_sub_path(self):
 
 
 class NetXML(CAPXML):
-
     """
     class for capability whose type is net.
     """
-    __slots__ = ('interface', 'address')
+    # Example:
+    #<capability type='net'>
+    #<interface>eno1</interface>
+    #<address>44:37:e6:95:03:e4</address>
+    #<link speed='100' state='up'/>
+    #<feature name='rx'/>
+    #<feature name='tx'/>
+    #<capability type='80203'/>
+    #</capability>
+    #</device>
+
+    __slots__ = ('interface', 'address', 'link_speed', 'link_state')
 
     def __init__(self, virsh_instance=base.virsh):
         accessors.XMLElementText('interface', self, parent_xpath='/',
                                  tag_name='interface')
         accessors.XMLElementText('address', self, parent_xpath='/',
                                  tag_name='address')
+        accessors.XMLAttribute('link_speed', self, parent_xpath='/',
+                               tag_name='link', attribute='speed')
+        accessors.XMLAttribute('link_state', self, parent_xpath='/',
+                               tag_name='link', attribute='state')
+        super(NetXML, self).__init__(virsh_instance=virsh_instance)
+        self.xml = (' <capability type=\'net\'></capability>')
+
+    __key2filename_dict__ = {'address': 'address',
```

Comment:
I assume miss interface attribute

Reply:
Yes, but I am not use this attribute for check, so at first I delete this attribute.